suite-native/biometrics/src/useBiometricsSettings.ts

```diff
@@ -0,0 +1,77 @@
+import { useCallback } from 'react';
+
+import { useAlert } from '@suite-native/alerts/src';
+import { analytics, EventType } from '@suite-native/analytics';
+
+import {
+    useIsBiometricsEnabled,
+    useIsBiometricsOverlayVisible,
+    useIsUserAuthenticated,
+} from './biometricsAtoms';
+import { getIsBiometricsFeatureAvailable } from './isBiometricsFeatureAvailable';
+import { authenticate } from './useBiometrics';
+
+export enum BiometricsToggleResult {
+    Enabled,
+    Disabled,
+    Failed,
+    NotAvailable,
+}
```

Comment:
Avoid using enums
https://www.notion.so/satoshilabs/Typescript-4640c944a60442e684b45f343104de37?pvs=4

Reply:
changed in c96f479